src/Dispatcher.php

```diff
@@ -65,7 +65,7 @@ public function __invoke(RequestInterface $request)
                 $function = (new ReflectionClass($controller))->getShortName();
 
                 $eventName = sprintf('%s.%s', strtolower($function), $method);
```

Comment:
This could be replaced with 

```php
$function = strtolower((new ReflectionClass($controller))->getShortName());
$eventName = "{$function}.{$method}"; // this improved ~1us
```

TIME | Total        | Single
--------|--------------|--------------------
TIME | 1.500784 | 0.015008 [ms]  
TIME | 1.411284 | 0.014113 [ms]

(no reason) :smile: 

I have also tried with a more ugly version :smile_cat: :

```php
$function = strtolower(basename(str_replace("\\", "/", get_class($controller))));
```

But there is no performance improvements